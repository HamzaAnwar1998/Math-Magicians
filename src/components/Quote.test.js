import React from 'react';
import renderer from 'react-test-renderer';

test('Quote should match this text', () => {
  const tree = renderer.create(
    <p>
      Mathematics is not about numbers, equations, computations
      or algorithms. It is about understanding. -- William Paul Thurston
    </p>,
  );
  expect(tree).toMatchSnapshot();
});
