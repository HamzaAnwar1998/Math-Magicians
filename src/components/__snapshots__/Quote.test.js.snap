// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Quote should match this text 1`] = `
<p>
  Mathematics is not about numbers, equations, computations or algorithms. It is about understanding. -- William Paul Thurston
</p>
`;
